test/services/helper/helper.spec.js

```diff
@@ -17,14 +17,16 @@ describe("helper functionality", function () {
       const mockWorkflowObject = require("../../dataset/create.json");
       const mockFields = ["sky"];
       const result = helper.objectContainsFields(mockWorkflowObject, mockFields);
-      expect(result).to.be.false;
+      expect(result).to.be.a("string");
```

Comment:
check that unit tests it("message") defines the expected results properly

Reply:
fixed in 7ad3409